README.md

```diff
@@ -30,7 +30,7 @@ Please see the latest [release note](https://github.com/ACINQ/eclair/releases) f
 
 ### Configuring Bitcoin Core
 
-:warning: Eclair requires Bitcoin Core 0.16.0 or higher. If you are upgrading an existing wallet, you need to create a new address and send all your funds to that address.
+:warning: Eclair requires Bitcoin Core 0.16.3 or higher. If you are upgrading an existing wallet, you need to create a new address and send all your funds to that address.
 
 Eclair needs a _synchronized_, _segwit-ready_, **_zeromq-enabled_**, _wallet-enabled_, _non-pruning_, _tx-indexing_ [Bitcoin Core](https://github.com/bitcoin/bitcoin) node. 
 Eclair will use any BTC it finds in the Bitcoin Core wallet to fund any channels you choose to open. Eclair will return BTC from closed channels to this wallet.
```

Comment:
The default `bitcoin.conf` needs to be amended because 0.17.0 introduced config sections. By default some parameters like `rpcuser` only apply to mainnet if no section is provided. See the [release notes](https://bitcoincore.org/en/releases/0.17.0/).

Reply:
Hmm on 2nd thought it should work, but I still think adding a `[testnet]` section like I did for the integration tests conf file would save users some headaches.

Reply:
I don't think we should add the changes to use 0.17.0 in our integration tests in this PR, my intention was to have something that is clean and that people can use right away with 0.17.0 and 0.16.3, while we work on changing our RPC calls (AFAIU 0.18.0 won't have `signrawtransaction` anymore)
  
Yes I will add a few words about the new sections as soon as I figure out how to use them properly :)